src/control/common/file_utils.go

```diff
@@ -219,3 +221,9 @@ func SyncDir(path string) (err error) {
 
 	return d.Sync()
 }
+
+// CheckSudo returns true if current process is running under sudo.
+func CheckSudo() (bool, string) {
+	usr := os.Getenv(sudoUserEnv)
```

Comment:
If the user is actually logged in as root (not a sudoer), I don't think SUDO_USER will be set.

Reply:
yes that's true, but that's what we want (and the error message will reflect it) because otherwise we have to manually set the target user. do we want to have to explicitly set the target user?